### DISCUSSION TITLE
Definicion de constantes

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#95] [Preguntas y respuestas] [answered]
hola! estoy empezando a encarar los ejercicios de cadenas seguras y mientras estaba buscando info sobre la definicion de constantes me encontre con una manera de crear conjuntos de constantes  usando 'enum' en una variable con las constantes dentro. hay alguna diferencia entre usar `enum ejemplo{constantes}` y usar `#define constante`? y si hay una diferencia ¿podemos usarla en el ejercicio?

**Comment by martinvilu** [comment, marked as answer]
Y cuando vimos enums? 🤭

**Reply by rinrot** [reply]
okok, no me adelanto entonces
